code/game/objects/items/weapons/trick.dm

```diff
@@ -0,0 +1,661 @@
+///////////////////Trick Syndicate Deck of Cards/////////////////
```

Comment:
You've not included trick.dm as a tgstation.dme addition.

Have you even tested this?